src/gatherers/service-worker.js

```diff
@@ -20,34 +20,46 @@ const Gather = require('./gather');
 
 class ServiceWorker extends Gather {
   get name() {
-    return 'serviceWorkers';
+    return 'serviceWorker';
   }
 
   reloadSetup(options) {
     const driver = options.driver;
-    this.resolved = false;
 
-    this.artifactsResolved = new Promise((res, _) => {
-      driver.on('ServiceWorker.workerVersionUpdated', data => {
-        if (!this.resolved) {
-          const controlledClients =
-              ServiceWorker.getActivatedServiceWorker(data.versions, options.url);
+    this.artifactsResolved = new Promise((resolve, reject) => {
```

Comment:
let's redo this thing to mimic our `getSecurityState` style.

so it'll be like `getServiceWorkerVersions`